### PR TITLE
Improve sending/retrieving/swarm logs

### PR DESCRIPTION
**Pull request by BeaudanBrown**
Cleaned up a lot of the logs, stop them from printing more than once etc
Might need some more cleaning later but want to get this in to get a better feel for them before making more changes

### CODE REVIEW

File: js/modules/loki_snode_api.js [hunk 1]
```diff
@@ -233,25 +232,15 @@ class LokiSnodeAPI {
       result = await response.text();
     }
 
-    if (response.status >= 0 && response.status < 400) {
-      return result.nodes;
+    if (response.status !== 200 || !result.nodes || result.nodes === []) {
```
Comment (Mikunj) [inline js/modules/loki_snode_api.js]: you can do `!response.ok` instead of checking for `200`

File: libtextsecure/errors.js [hunk 2]
```diff
@@ -127,18 +127,14 @@
...
     this.number = number.split('.')[0];
 
     ReplayableError.call(this, {
       name: 'EmptySwarmError',
-      message: 'Could not get any swarm nodes to query',
+      message,
     });
-
-    if (error) {
-      appendStack(this, error);
```
Comment (sachaaaaa) [inline libtextsecure/errors.js]: Any idea what this line was for?
Do we not call this anymore then?

Comment (BeaudanBrown) [reply libtextsecure/errors.js]: Could probably change this to take an error and a message, the problem was that currently there wouldn't be any information about *where* the empty swarm error was occurring (i.e. we couldn't get any to begin with or we couldn't connect to any of the nodes we got in our list)
The error was usually just like a timeout or connection error, could still be useful tho I guess

Comment (sachaaaaa) [reply libtextsecure/errors.js]: I take your word for it that we can afford to ditch that :)

**Pull request merged**